andela/thor-ah-frontend
# #161419917 Bug fixes

#### What does this PR do?

-- Add pagination component to the categories page and favorite article page
-- Display list of articles on the categories page
-- Fix the all articles view in accordance to feedback
-- Replace highlight and comment icon with a better icon

#### How should this be manually tested?

-- Clone the repo from the latest change.
-- Install node.js on your local machine.
-- Clone the backend repo from [here](https://github.com/andela/thor-ah)
-- update the `.env` file on the backend according to the ```.env-sample```
-- Run ```npm install```
-- Run npm run dev.
-- Visit the landing page to see the new display for all articles
-- Visit the one article page, highlight a text and see the new icon
-- Visit the category page and see articles paginated if more than 12 articles are returned
-- Visit the favorite article page and see articles paginated if more than 6 articles are returned

#### What are the relevant pivotal tracker stories?

-- [#161419917](https://www.pivotaltracker.com/story/show/161419917)

#### Screenshots (if appropriate)
<img width="177" alt="screen shot 2018-10-26 at 05 24 19" src="https://user-images.githubusercontent.com/26516851/47544550-78c0ad80-d8df-11e8-9ba2-43be3d72b6f0.png">
